app/components/app-notification.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
```

Comment:
Use ES6 destructuring. (See other component files to know how `Component` is defined.